igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/gitlab/client/GitLabMaster.groovy

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 bol.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm.gitlab.client
+
+import groovy.util.logging.Slf4j
+
+/**
+ * Wrapper class for a collection of GitLab clients
```

Comment:
Can this ever be a collection? It looks like a single gitlab integration can be defined in the configuration. It looks more like Wrapper for the gitlab client.

Reply:
Also copied from the GitHub variant.